cobc/scanner.l

```diff
@@ -1271,11 +1271,67 @@ error_literal (const char *type, const char *literal)
 	cb_error ("%s", err_msg);
 }
 
+/* EBCDIC to ASCII table, used in read_literal */
+static const cob_u8_t cob_ebcdic_ascii[256] = {
```

Comment:
It would make sense to add a comment saying that this table is the same as generated at codegen.c:3195 by `output_ebcdic_to_ascii_table`. Also, why not use the one generated by `output_full_ebcdic_table` ? What's the difference ?

Reply:
Actually there are many more ways this should be done, but first I went for the obvious one: The standard says "At object time, each symbolic-character in the non-numeric literal is replaced by the character from the native character set
it represents." I went straight to ASCII as native character set for simplicity.
Now that you mention it I just found the macro to support both ASCII and EBCDIC native character sets…